Mlem/App/Views/Root/Tabs/Settings/DeveloperSettingsView.swift

```diff
@@ -28,6 +34,28 @@ struct DeveloperSettingsView: View {
                 NavigationLink(String("Error Log"), destination: .settings(.errorLog))
             }
             
+            Section {
+                HStack {
+                    Text("Status")
+                    Spacer()
+                    if let backendStatus {
+                        Image(systemName: Icons.present)
+                            .foregroundStyle(backendStatus ? .themedPositive : .themedNegative)
+                    } else {
+                        ProgressView()
+                    }
+                }
+                
+                Button("Refresh") { checkBackendStatus() }
+            } header: {
+                Text("Backend")
+            } footer: {
+                if let lastBackendStatusCheck {
+                    Text("Refreshed \(lastBackendStatusCheck.formatted(date: .abbreviated, time: .standard))")
+                }
```

Comment:
*nit*

When you open the page this text takes a second to load, which is a little jarring - this could be fixed with something like

```suggestion
                if let lastBackendStatusCheck {
                    Text(verbatim: "Refreshed \(lastBackendStatusCheck.formatted(date: .abbreviated, time: .standard))")
                } else {
                    Text(verbatim: "Refreshing...")
                }
```

Then again this is the developer settings page so it doesn't really matter lol